include/package-pack.mk

```diff
@@ -205,6 +205,8 @@ $(_endef)
 	rm -rf $$(IDIR_$(1))
 ifeq ($$(CONFIG_USE_APK),)
 	$$(call remove_ipkg_files,$(1),$$(call opkg_package_files,$(call gen_package_wildcard,$(1))))
+else
+	$$(call remove_ipkg_files,$(1),$$(call apk_package_files,$(call gen_package_wildcard,$(1))))
```

Comment:
This is fine with me, however we could keep in mind that APK can actually handle multiple versions of the same package and eventually support that.

Reply:
Can this be a problem for stable release where we update some package not marked as hold?

Reply:
I guess, let's stick with what we know for now and please ignore my previous comment.